packages/web-runtime/src/store/user.js

```diff
@@ -99,6 +101,14 @@ const actions = {
         const userGroups = await client.users.getUserGroups(login.id)
         const user = await client.users.getUser(login.id)
 
+        // FIXME: Can be removed as soon as the uuid is integrated in the OCS api
+        const graphClient = clientService.graphAuthenticated(instance, token)
```

Comment:
Found one more nitpick: sufficient to declare the graphClient within the `if` clause.